### ISSUE TITLE
Cannot see the settings

### ISSUE THREAD

**Issue by frynio** [Glavin001/atom-beautify#1601]
# Description

Under Settings/Packages/atom-beautify/Settings there is only General tab, no languages here

# Steps to Reproduce

1. Go to Settings/Packages/atom-beautify/Settings

# Debug

Here is a link to the `debug.md` Gist: <INSERT GIST HERE>

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue
- [x] Uninstalled and installed Atom again (with deleting .atom folder)

debug: https://gist.github.com/frynio/743b72b785655c61bd38bfc11f261eeb

**Comment by prettydiff** [comment]
Does this problem only occur for C++ code or is the problem persistent for a variety of languages?

**Comment by frynio** [comment]
For every language I work with (f.e golang). I mean it can beautify, cuz it will find gofmt I guess, it can beautify C++ code (with Beautify shortcut), cuz it can find uncrustify. But I cannot set Beautify on save, or configs path

**Comment by Glavin001** [comment]
I think I broke something with https://github.com/Glavin001/atom-beautify/pull/1598

@frynio could you check for `atom-beautify/src/options.json` file in the Atom package's install path?

For example, on my Mac:

![image](https://cloud.githubusercontent.com/assets/1885333/25067846/2fd379f2-2226-11e7-87d1-c48337167da3.png)


```
/Users/glavin/.atom/packages/atom-beautify/src:
beautifiers     beautify.coffee config.coffee   languages       logger.coffee   options.json    views
```

**Comment by frynio** [comment]
[frynio@manjaro atom-beautify]$ ls
appveyor          CHANGELOG.md     docs               keymaps     node_modules              README.md           script  styles
appveyor.yml      coffeelint.json  examples           LICENSE.md  package.json              README-template.md  spec
build-package.sh  CONTRIBUTING.md  ISSUE_TEMPLATE.md  menus       PULL_REQUEST_TEMPLATE.md  ROADMAP.md          src
[frynio@manjaro atom-beautify]$ ls src
beautifiers  beautify.coffee  config.coffee  languages  logger.coffee  views
[frynio@manjaro atom-beautify]$ 

No options.json here

**Comment by Glavin001** [comment]
I can also confirm `options.json` is not here when I do a clean install. Thanks for clarifying. I think there was an issue when publishing to Atom's package registry.

**Comment by frynio** [comment]
Okay. So after you fix it, should I just reinstall package?

**Comment by Glavin001** [comment]
Yup, it was an issue with publishing using `apm` which acts differently than `npm`: https://discuss.atom.io/t/how-to-use-the-npmignore/9208/11

**Comment by frynio** [comment]
Great, thank you! :D

**Comment by Glavin001** [comment]
Published to v0.29.22

Thank you for letting me know! This would have caused a lot of problems for a lot of people.

**Comment by Glavin001** [comment]
Please let me know if this problem is resolved for you 😃 .
